src/chrome/content/rules/Bukalapak.com.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="Bukalapak.com">
+	<target host="bukalapak.com" />
+	<target host="www.bukalapak.com" />
+	<target host="agen.bukalapak.com" />
+	<target host="careers.bukalapak.com" />
+	<target host="panduan.bukalapak.com" />
+	<target host="s0.bukalapak.com" />
+	<target host="s1.bukalapak.com" />
+	<target host="s2.bukalapak.com" />
+	<target host="s3.bukalapak.com" />
+	<target host="s4.bukalapak.com" />
```

Comment:
Please also add:
```
komunitas.bukalapak.com
penggerak.bukalapak.com

Reply:
Okay. I have added it and I have updated other PRs